lib/backend/k8sv2/custom/bgpconfig.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package custom
+
+import (
+	"encoding/json"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/projectcalico/libcalico-go/lib/apiv2"
+)
+
+type BGPConfig struct {
```

Comment:
Can we now just use the definitions in apiv2 for all these things?

Reply:
same is true now for all of the CRD backed resources - i think the apiv2 definitions should be "k8s" crd compliant.